mavros_extras/CMakeLists.txt

```diff
@@ -109,7 +110,7 @@ install(PROGRAMS
 )
 
 ## Mark executables and/or libraries for installation
-install(TARGETS mavros_extras copter_visualization servo_state_publisher
+install(TARGETS mavros_extras visualization servo_state_publisher
```

Comment:
Do not rename node. No reason. That still models simple quadcopter, not plane or anything else.